.github/workflows/cypress-testing.yml

```diff
@@ -330,6 +330,19 @@ jobs:
           SERVER_LOGGER_LEVEL: debug
           SERVER_LOGGER_FORMAT: console
 
+      azurite:
```

Comment:
Azurite is a local emulator for blob/table/queue storage. We could probably start it only in assessment mode though (maybe another flag for the run app dependencies?)